ext/ddtrace.c

```diff
@@ -141,10 +140,10 @@ static ZEND_INI_MH(dd_OnUpdateDisabled) {
 }
 
 PHP_INI_BEGIN()
-    ZEND_INI_ENTRY("ddtrace.disable", "0", PHP_INI_SYSTEM, dd_OnUpdateDisabled)
+    ZEND_INI_ENTRY("datadog.trace.disable", "0", PHP_INI_SYSTEM, dd_OnUpdateDisabled)
```

Comment:
It makes sense, but this is not just dropping the deprecation and should be loudly noted with the 1.0 changes.

Reply:
Oops I committed this without wanting 😬 I was starting to do [AIT-2131](https://datadoghq.atlassian.net/browse/AIT-2131) but instead moved to smthg else (system tests) and forgot not to push this 🤦 Either way, let's do it in another PR

Reply:
The same goes for `datadog.trace.cgroup_file` and the changes in startup_logging.c